### DISCUSSION TITLE
语汇字音地图

### DISCUSSION THREAD

**Discussion by Guanchishan** [MindongLab/yngdieng#530] [产品功能 ideas]
**你提出的功能想解决什么问题**
显示内容：
- 给定字词的使用分布地图
- 该字词在不同地区的变体的地图
- 一个字词下不同读音的使用分布地图

地图显示方式的商榷：
- 一抹色（比如词汇地域标为闽清则闽清全域上色，边界处渐变至无色，不把边界画实，如果有人反应都里不这么读，再把都里挖掉，边界线同样做渐变至无色处理）？

> 山月：这样能体现语言分布是过渡的这一思想。但是这样可能会产生大量不讲某词却被标上大区颜色的情况发生，大头钉至少回避了犯这种错的机会。

- 大头钉📍？

> 子善（会怕词汇分布情况犬牙交错的话）可以把小小地图里的大头钉做大点

> 山月：或者各县市级大头钉做大，各具体地方大头钉做小小的？这样如果连江有A词，连江某地却有B词，某某地却有C词的话，显示于地图上也不至于太乱

**他山之石**
[粤语词典](https://www.jyutdict.org/)：
![image](https://user-images.githubusercontent.com/30859354/90679010-214cb800-e292-11ea-8cb4-52687b864af9.png)

**你觉得理想的解决方案是什么**

请参与下方的讨论。结果将更新于此。

**其他信息**
由子善（@ZySieng）提案于2020.8.15榕典、输入法例会，山月（@guanchishan）整理。

**Comment by ztl8702** [comment]
@Guanchishan  你画的草图可以发上来

**Comment by Guanchishan** [comment]
> @Guanchishan  你画的草图可以发上来

如果以📍方式呈現，頁面效果如圖![image](https://user-images.githubusercontent.com/30859354/90719690-a1e9d380-e2e7-11ea-94ee-92227cf87df0.jpeg)
